### PR TITLE
Batches route

### PR DESCRIPTION
**Pull request by TimothyAL**

### CODE REVIEW

File: src/data/managers_db.py [hunk 1]
```diff
@@ -1,5 +1,7 @@
-from src.data.data import DatabaseConnection
+'''this module is to store the calls to the manager collection'''
+import pymongo
```
Comment (rlayneorr) [inline src/data/managers_db.py]: Why are you importing this?

File: src/router/employees.py [hunk 2]
```diff
@@ -78,7 +78,7 @@ def get(self, user_id):
 
     @api.doc(body=swot_fields)
     @api.response(200, 'Status code of response')
-    def put(self, user_id):
+    def post(self, user_id):
         '''Function for handling PUT /employees/user_id requests'''
```
Comment (rlayneorr) [inline src/router/employees.py]: Comment is incorrect.

**Review by rlayneorr — CHANGES_REQUESTED** [review]

File: src/data/managers_db.py (hunk 1 above)
Comment (TimothyAL) [reply src/data/managers_db.py]: not sure why it reverted it will be taken care of with next commit

File: src/router/employees.py (hunk 2 above)
Comment (TimothyAL) [reply src/router/employees.py]: Yes it is. It will be fixed on the next commit.

**Review by rlayneorr — APPROVED** [review]

**Pull request merged**